src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -382,11 +382,14 @@ public static void ShowDialog(View content = null)
             var window = NUIApplication.GetDefaultWindow();
             var defaultNavigator = window.GetDefaultNavigator();
 
-            var dialog = new Dialog(content);
-            SetDialogScrim(dialog);
-
-            var dialogPage = new Page(dialog);
-            defaultNavigator.Push(dialogPage);
+            using (var dialog = new Dialog(content))
```

Comment:
using(){ } should not be used in this case since the constructor of Page and Navigator.Push do not clone their parameters.